test/run

```diff
@@ -117,21 +117,21 @@ function run_change_password_test() {
   # Create Redis container with persistent volume and set the initial password
   create_container "testpass1" -e REDIS_PASSWORD=foo \
     -v ${tmpdir}:/var/lib/redis/data:Z
-  check_result $?
+  ct_check_testcase_result $?
   test_connection testpass1 foo
-  check_result $?
-  docker stop $(get_cid testpass1) >/dev/null
+  ct_check_testcase_result $?
+  docker stop $(ct_get_cid testpass1) >/dev/null
 
   # Create second container with changed password
   create_container "testpass2" -e REDIS_PASSWORD=bar \
     -v ${tmpdir}:/var/lib/redis/data:Z
-  check_result $?
+  ct_check_testcase_result $?
   test_connection testpass2 bar
-  check_result $?
+  ct_check_testcase_result $?
   # The old password should not work anymore
-  container_ip="$(get_container_ip testpass2)"
-  connection_works "$container_ip" bar
-  check_result $?
+  container_ip="$(ct_get_cip testpass2)"
+  ! connection_works "$container_ip" foo
```

Comment:
This line also seem not connected with test-lib functions usage. Could it be placed in different commit?

Reply:
yep, will do